addons/website_sale/static/src/js/website_sale_tour.js

```diff
@@ -0,0 +1,395 @@
+                    model: 'res.config.settings',
+                    method: 'execute',
+                    args: [[resId]],
+                });
+            });
+            return Promise.all([taxDef, def2]).then(function (res) {
+                window.location.href = '/web/session/logout?redirect=/shop?search=Storage Box';
+            });
+        },
+    },
+    // Testing b2c with Tax-Excluded Prices
+    {
+        content: "Open product page",
+        trigger: '.oe_product_cart a:contains("Storage Box")',
+    },
+    {
+        content: "Add one more storage box",
+        trigger: '.js_add_cart_json:eq(1)',
+    },
+    {
+        content: "Check b2b Tax-Excluded Prices",
+        trigger: '.product_price .oe_price .oe_currency_value:containsExact(79.00)',
+    },
+    {
+        content: "Click on add to cart",
+        trigger: '#add_to_cart',
+    },
+    {
+        content: "Check for 2 products in cart and proceed to checkout",
+        extra_trigger: '#cart_products tr:contains("Storage Box") input.js_quantity:propValue(2)',
+        trigger: 'a[href*="/shop/checkout"]',
+    },
+    {
+        content: "Check Price b2b subtotal",
+        trigger: 'tr#order_total_untaxed .oe_currency_value:containsExact(158.00)',
+    },
+    {
+        content: "Check Price b2b Sale Tax(15%)",
+        trigger: 'tr#order_total_taxes .oe_currency_value:containsExact(23.70)',
+    },
+    {
+        content: "Check Price b2b Total amount",
+        trigger: 'tr#order_total .oe_currency_value:containsExact(181.70)',
+    },
+    {
+        content: "Fulfill address form",
+        trigger: 'select[name="country_id"]',
+        run : function () {
+            $('input[name="name"]').val('abc');
+            $('input[name="phone"]').val('99999999');
+            $('input[name="email"]').val('abc@odoo.com');
+            $('input[name="street"]').val('1,C.G Road');
+            $('input[name="city"]').val('xyz');
+            $('#country_id option:eq(1)').attr('selected', true);
+        },
+    },
+    {
+        content: "Shipping address is not same as billing address",
+        trigger: '#shipping_use_same',
+    },
+    {
+        content: "Click on next button",
+        trigger: '.oe_cart .btn:contains("Next")',
+    },
+    {
+        content: "Fulfill shipping address form",
+        trigger: 'select[name="country_id"]',
+        extra_trigger: 'h2:contains("Shipping Address")',
+        run : function () {
+            $('input[name="name"]').val('def');
+            $('input[name="phone"]').val('8888888888');
+            $('input[name="street"]').val('2,xyz');
+            $('input[name="city"]').val('xyz');
+            $('#country_id option:eq(1)').attr('selected', true);
+        },
+    },
+    {
+        content: "Click on next button",
+        trigger: '.oe_cart .btn:contains("Next")',
+    },
+    {
+        content: "Click for edit address",
+        trigger: 'a:contains("Edit") i',
+    },
```

Comment:
Could you please add the step that change the address not just click on edit?

Reply:
👍